src/pretalx/orga/forms/cfp.py

```diff
@@ -421,7 +447,18 @@ class ReminderFilterForm(QuestionFilterForm):
     )
 
     def get_question_queryset(self):
-        return Question.objects.filter(
+        questions = Question.objects.all()
+        now = datetime.datetime.now()
+        # We want to exclude questions with "freeze after", the deadlines of which have passed
+        questions_frozen = set()
```

Comment:
This should be doable in a single query, without looping, I think. Something like this:

```python
Question.objects.filter(
    event=self.event,
    target__in=["speaker", "submission"],
).exclude(Q(question_required="freeze after") & Q(deadline__lt=now()))
```